app/controllers/Utils.scala

```diff
@@ -13,12 +13,14 @@ object Utils {
   /**
    * Return base url given a request. This will add http or https to the front, for example
    * https://localhost:9443 will be returned if it is using https.
+   *
+   * Change type from Request[Any]
```

Comment:
Should this be deleted?

Reply:
When it was suggested we only have 1 version of the method, I left that in case any problems showed up later,
but these notes/etc should be enough of a record.  So I deleted it